.github/release/vote-email-template.md

```diff
@@ -6,73 +6,45 @@ This is a call for vote to release Apache Texera (incubating) ${VERSION}.
 
 == Release Candidate Artifacts ==
 
-The release candidate artifacts can be found at:
-https://dist.apache.org/repos/dist/dev/incubator/texera/${RC_DIR}/
+https://dist.apache.org/repos/dist/dev/incubator/texera/${VERSION}-RC${RC_NUM}/
 
-The artifacts include:
-- apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz (source tarball)
-- apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.asc (GPG signature)
-- apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.sha512 (SHA512 checksum)
+The directory contains:
+- Source tarball (.tar.gz)
+- GPG signature (.asc)
+- SHA512 checksum (.sha512)
 
 == Git Tag ==
 
-The Git tag for this release candidate:
-https://github.com/apache/incubator-texera/releases/tag/${TAG_NAME}
-
-The commit hash for this tag:
-${COMMIT_HASH}
-
-== Release Notes ==
-
-Release notes can be found at:
-https://github.com/apache/incubator-texera/releases/tag/${TAG_NAME}
+https://github.com/apache/texera/releases/tag/${TAG_NAME}
+Commit: ${COMMIT_HASH}
 
 == Keys ==
 
-The artifacts have been signed with Key [${GPG_KEY_ID}], corresponding to [${GPG_EMAIL}].
-
-The KEYS file containing the public keys can be found at:
-https://dist.apache.org/repos/dist/dev/incubator/texera/KEYS
-
-== How to Verify ==
-
-1. Download the release artifacts:
-
-   wget https://dist.apache.org/repos/dist/dev/incubator/texera/${RC_DIR}/apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz
-   wget https://dist.apache.org/repos/dist/dev/incubator/texera/${RC_DIR}/apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.asc
-   wget https://dist.apache.org/repos/dist/dev/incubator/texera/${RC_DIR}/apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.sha512
-
-2. Import the KEYS file and verify the GPG signature:
+The release was signed with GPG key [${GPG_KEY_ID}] (${GPG_EMAIL})
+KEYS file: https://downloads.apache.org/incubator/texera/KEYS
 
-   wget https://dist.apache.org/repos/dist/dev/incubator/texera/KEYS
-   gpg --import KEYS
-   gpg --verify apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.asc apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz
-
-3. Verify the SHA512 checksum:
-
-   sha512sum -c apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz.sha512
-
-4. Extract and build from source:
-
-   tar -xzf apache-texera-${VERSION}-rc${RC_NUM}-src.tar.gz
-   cd apache-texera-${VERSION}-rc${RC_NUM}-src
-   # Follow build instructions in README
-
-== How to Vote ==
+== Vote ==
 
 The vote will be open for at least 72 hours.
 
-Please vote accordingly:
-
 [ ] +1 Approve the release
 [ ]  0 No opinion
-[ ] -1 Disapprove the release (please provide the reason)
+[ ] -1 Disapprove the release (please provide reason)
+
+== Convenience Binaries ==
 
-== Checklist for Reference ==
+Docker images (tag: ${VERSION}):
+- https://hub.docker.com/r/apache/texera-workflow-compiling-service/tags?name=${VERSION}
```

Comment:
* you can't publish docker images with the version until after the release vote passes
* you can publish with the version and rc number before the vote for review
* please read https://incubator.apache.org/guides/distribution.html#docker

Reply:
fixed